### DISCUSSION TITLE
Как получить значок Galaxy Brain?

### DISCUSSION THREAD

**Discussion by AlexanderMarobe** [community/community#19433] [Other Feature Feedback, Questions, & Ideas] [answered]
Как получить значок Galaxy Brain?

**Comment by RobertGoodman08** [comment, marked as answer]
Ответьте как минимум на 2 обсуждения.
Обратите внимание, вы НЕ МОЖЕТЕ отмечать свои собственные ответы на свои собственные вопросы. Они должны быть отмечены другим пользователем.
Обсуждения должны быть в общедоступном хранилище.
В настоящее время некоторые пользователи наблюдают ошибку, из-за которой пометка ваших собственных ответов может помешать вам получить значок вместо этого.
Надеюсь, этот ответ помог вам, и всего наилучшего в получении достижения! :)

**Reply by Pavel8913** [reply]
thanks

**Comment by ryudongjae** [comment]
https://github.com/orgs/github-community/discussions/18542#discussioncomment-2974498

**Comment by 7iliama1** [comment]
👍🏼

**Comment by vag1ffm** [comment]
👍🏼

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @AlexanderMarobe. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by KeplerDE** [comment]
👍🏼
